clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -82,7 +82,7 @@ public void setup() throws Exception {
         sslClientConfigs = clientCertStores.getTrustingConfig(serverCertStores);
         this.channelBuilder = new SslChannelBuilder(Mode.CLIENT, null, false);
         this.channelBuilder.configure(sslClientConfigs);
-        this.selector = new Selector(5000, new Metrics(), new MockTime(), "MetricGroup", channelBuilder, new LogContext());
+        this.selector = new Selector(5000, 100, new Metrics(), new MockTime(), "MetricGroup", channelBuilder, new LogContext());
```

Comment:
Not sure why this was changed. This is the client-side selector. We are not delaying on clients? (Multiple occurrences)